src/middle-end/config.js

```diff
@@ -9,12 +9,14 @@ module.exports = {
     }),
     createStore: (reducer, { router }) => {
 
+        const middleware = [
+            MiddleEnd.middleware.thunk,
+            process.env.NODE_ENV !== 'test' && MiddleEnd.middleware.errorLogger,
```

Comment:
Do we want to include this in a production build?

Reply:
The more I thought about it, the less I saw an issue with it.  If we want to remove it from production, no problem, though!

Reply:
This was just my first thought.  After _actually_ thinking about it and looking at your docs for `errorLogger`, I don't have a problem with it.